thrift/shared.thrift

```diff
@@ -1258,6 +1258,7 @@ struct GetWorkflowExecutionHistoryRequest {
   50: optional bool waitForNewEvent
   60: optional HistoryEventFilterType HistoryEventFilterType
   70: optional bool skipArchival
+  80: optional bool rawHistorySupported
```

Comment:
nit: we might consider to name it just rawHistory and provide it as an option for client